docs/guide/alerts-guide.md

```diff
@@ -13,26 +13,28 @@ Receiving Alerts when your software is misbehaving or underperforming is crucial
 - **Criteria** that trigger **Alerts for monitored resources**
 - **Thresholds** defined for **Alerts** when selected **metrics reach certain severity levels**
-- **Notification channels** through which the responsible team will be notified about the incident
+- [**Notification channels**](https://sematext.com/docs/alerts/alert-notifications/) through which the responsible team will be notified about the incident
 
 [Sematext Alerts](../alerts) help you address underlying issues before they affect user experience. There are three types of Alerts. 
 
 - **Threshold** - Alerts that are based on classic thresholds. They are **triggered** when something **crosses a pre-defined threshold**.
+- **Heartbeat** - Alerts **triggered** when **something you are monitoring**, like your servers, containers, or your applications, **stops sending data** to Sematext Cloud.
 - **Anomaly** - Alerts based on statistical anomaly detection. They are **triggered** when **values** suddenly **change and deviate from the baseline**. 
-- **Heartbeat** - Alerts **triggered** when **something you are monitoring**, like your servers, containers, or your applications, **stops sending data** to Sematext Cloud. 
 
-**Threshold** and **Anomaly** Alerts can be triggered for both Monitoring and Logs Apps, while **Heartbeat** Alerts are only available for Monitoring Apps.
+![Sematext Anomaly Alerts](../images/guide/alerts-and-events/anomaly-alerts.png)
 
-Sematext lets you manage Alert Rules across your whole stack. You can use various notification channels to get alerted when critical issues occur so you can take action and resolve them. Our Alerts easily integrate with notification and chat software. 
+**Threshold** and **Anomaly** Alerts can be triggered for [Events](https://sematext.com/docs/events/) and [Monitoring](https://sematext.com/docs/alerts/creating-metrics-alerts/), [Logs](https://sematext.com/docs/alerts/creating-logs-alerts/), [Synthetics](https://sematext.com/docs/alerts/creating-synthetics-alerts/) and [Experience](https://sematext.com/docs/alerts/creating-experience-alerts/) Apps, while [**Heartbeat**](https://sematext.com/docs/alerts/creating-heartbeat-alerts/) Alerts are only available for Monitoring Apps.
 
-- PagerDuty
-- OpsGenie
-- HipChat
-- Slack, and [many more](https://sematext.com/docs/integration/). 
+Sematext lets you manage Alert Rules across your whole stack. You can use various [notification channels](https://sematext.com/docs/alerts/alert-notifications/) to get alerted when critical issues occur so you can take action and resolve them. Our Alerts easily integrate with notification and chat software. 
+
+- [PagerDuty](https://sematext.com/docs/integration/alerts-pagerduty-integration/)
+- [OpsGenie](https://sematext.com/docs/integration/alerts-opsgenie-integration/)
+- [HipChat](https://sematext.com/docs/integration/alerts-hipchat-integration/)
```

Comment:
We can replace this with something.  HipChat is history. https://community.atlassian.com/t5/Hipchat/ct-p/hipchat
Maybe we should remove it all together.

Reply:
Removed, too many links there anyways.